packages/components/src/ClickableStyle/ClickableStyle.tsx

```diff
@@ -22,7 +22,7 @@ export type ClickableStyleProps<IComponent extends React.ElementType> = {
   /**
    * The style of the element.
    */
-  variant: "flat" | "outline" | "link";
+  variant: "flat" | "outline" | "text" | "link";
```

Comment:
> We should probably make the types restrictive here so it's not possible to create a "text" button with no icon.

Agree -- let's either make a TODO here or be sure to land after the icon change.

I'm also wondering if `"plain"` might be more descriptive than `"text"`? these variant names are already different from the UI Kit groupings

Reply:
Sounds good – I found "text" really confusing before because links are also just text. "Plain" makes sense because links are underlined whereas this variant has no decoration (aside from the icon but any of the buttons can have an icon).